### ISSUE TITLE
$navigateTo() frame ID issue

### ISSUE THREAD

**Issue by unremarkablegarden** [nativescript-vue/nativescript-vue-navigator#55]
To use this.$navigateTo() you now have to specify the frame ID 'navigator' explicitly or it won't navigate. Also won't give an error unless you add a catch().

`this.$navigateTo(ItemDetails, {
                frame: 'navigator',
                props: {
                    item: post,
                    animated: true,
                    type: 'news' 
                }
            }).catch(error => console.log(error))`

**Comment by rigor789** [comment]
That's correct - you shouldn't need to use `$navigateTo` with the navigator, the call should be

```js
this.$navigator.navigate('/path/of/the/details/route', {
    props: { ... }
}).catch(error => console.log(error))
```
